hooks/use-Get-Payment-Status.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+
+import { useSubscription } from "@apollo/client"
+
+import { OperationError } from "../lib/graphql/index.types.d"
+import { LN_INVOICE_PAYMENT_STATUS } from "../lib/graphql/subscription"
+
+interface Props {
+  paymentRequest: string | undefined
+  callbackFn?: () => void
+}
+
+const useGetPaymentStatus = ({ paymentRequest, callbackFn }: Props) => {
```

Comment:
Should be named to reflect it is just for LN payments

Reply:
Okay